backend/main.py

```
@@ -82,79 +92,93 @@ async def upload_drawing(file: UploadFile = File(...)):
+    """Поиск через векторного агента"""
     async with httpx.AsyncClient(timeout=15.0) as client:
         try:
             response = await client.post(f"{AGENT_URL}/search", json={"query": q, "limit": limit})
             response.raise_for_status()
             return response.json()
-        except httpx.HTTPStatusError as e:
-            raise HTTPException(
-                status_code=e.response.status_code,
-                detail=f"Agent error (HTTP {e.response.status_code}): {e.response.text}"
-            )
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)